webpack/assets/javascripts/react_app/components/StatisticsChartsList/StatisticsChartsListActions.js

```diff
@@ -1,11 +1,11 @@
-import { STATISTICS_DATA } from '../../consts';
-import { get } from '../../API';
+import { STATISTICS_DATA } from './StatisticsChartsListConsts';
+import { get } from '../../redux/API';
 
 export const getStatisticsData = charts => dispatch => {
   charts.forEach(chart => {
     dispatch(
       get({
-        key: STATISTICS_DATA,
+        key: `${STATISTICS_DATA}_${chart.id}`,
```

Comment:
@laviro not sure if I should use `toUpperCase` on the `chart.id`, what do you think?
a key for example: `STATISTICS_DATA_operatingsystem`

Reply:
imo it doesn't matter, can you declare it to the `index` file as well and declare it once?
than you can pass it to the get action and to the selectors.